client/components/telecom/telephony/group/number/feature/ovhPabx/menu/edit/telephony-group-number-feature-ovh-pabx-menu-edit.html

```diff
@@ -294,11 +290,11 @@ <h2 data-ng-if="['greetSound', 'invalidSound', 'greetSoundTts', 'invalidSoundTts
                         <em data-translate="telephony_number_feature_ovh_pabx_step_rule_no_tts_added"></em>
                     </div>
 
-                    <label class="section-item item-choice full-width pointer dtable table-fixed"
+                    <label class="section-item item-choice w-100 d-table table-fixed"
```

Comment:
table-fixed should be removed no ?

@antleblanc ?

Reply:
In this situation we can get rid of `table-fixed`.